plugins/rect/README.md

```diff
@@ -1,33 +1,33 @@
 # rect
 
-This plugin adds rect (width + height) sizing utilities to Tailwind.
+This plugin adds rect (width + height) sizing utilities to Tailwind, which requires the rem function from [FedPack's Tailwind config](https://github.com/vigetlabs/FedPack/blob/master/tailwind.config.js).
```

Comment:
Hmm, now that this is installable I dont love the requirement of passing a function -- seems better to just recreate the rem function?